CHANGELOG.md

```diff
@@ -0,0 +1,4 @@
+v1.0.0
+-----
+
+* **Breaking Change:** Will not configure or load the `anytv-i18n` module anymore.
```

Comment:
I guess this is to make sure i18n module can be re-used? Where did you observe where i18n is loaded multiple times?